common/changes/office-ui-fabric-react/tooltip_2019-02-07-23-13.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "packageName": "office-ui-fabric-react",
+      "type": "none"
```

Comment:
Do we want "patch" here when altering example files? @ecraig12345 said the following in [another PR](https://github.com/OfficeDev/office-ui-fabric-react/pull/7905)

> Since example changes affect the website, it's probably best to use change type patch.

Reply:
oh i suppose that is needed for fabric website to pick this up...